assets/wizards/advertising/views/ad-units/index.js

```diff
@@ -5,38 +5,167 @@
+						<Notice
+							noticeText={ __(
+								'Your GAM network code is different than the network code the site was configured with. Editing has been disabled.',
+								'newspack'
+							) }
+							isError
+						/>
+					) }
+					{ gamConnectionStatus?.connected === false && (
+						<Notice
+							noticeText={ gamConnectionMessage || warningNoticeText }
+							isWarning={ ! gamConnectionMessage }
+							isError={ gamConnectionMessage }
+						/>
+					) }
+				</>
+			) : (
+				<div className="flex items-end" style={ { margin: '-30px 0' } }>
+					<TextControl
+						className="mr2"
+						label={ __( 'Network code', 'newspack' ) }
+						value={ networkCode }
+						onChange={ setNetworkCode }
+					/>
+					<div className="mb4">
+						<Button onClick={ saveNetworkCode } isPrimary>
+							{ __( 'Save', 'newspack' ) }
+						</Button>
+					</div>
+				</div>
+			) }
 			<p>
 				{ __(
-					'Set up multiple ad units to use on your homepage, articles and other places throughout your site. You can place ads through our Newspack Ad Block in the Editor.'
+					'Set up multiple ad units to use on your homepage, articles and other places throughout your site.'
+				) }
+				<br />
+				{ __(
+					'You can place ads through our Newspack Ad Block in the Editor, Newspack Ad widget, and using the global placements.'
 				) }
 			</p>
-			{ Object.values( adUnits ).map( ( { id, name } ) => {
-				return (
-					<ActionCard
-						key={ id }
-						title={ name }
-						actionText={ __( 'Edit' ) }
-						titleLink={ `#${ service }/${ id }` }
-						href={ `#${ service }/${ id }` }
-						secondaryActionText={ __( 'Delete' ) }
-						onSecondaryActionClick={ () => onDelete( id ) }
-					/>
-				);
-			} ) }
-		</>
+			<Card noBorder>
+				{ Object.values( adUnits )
+					.sort( ( a, b ) => b.name.localeCompare( a.name ) )
+					.sort( a => ( a.is_legacy ? 1 : -1 ) )
+					.map( adUnit => {
+						const editLink = `#${ service }/${ adUnit.id }`;
+						const isDisabled = adUnit.is_legacy
+							? false === gamConnectionStatus?.is_network_code_matched
+							: false;
+						const buttonProps = {
+							disabled: isDisabled,
+							isQuaternary: true,
+							isSmall: true,
+							tooltipPosition: 'bottom center',
+						};
+						return (
+							<ActionCard
+								disabled={ isDisabled }
+								key={ adUnit.id }
+								title={ adUnit.name }
+								isSmall
+								titleLink={ isDisabled ? null : editLink }
+								className="mv0"
+								{ ...( adUnit.is_legacy
+									? {}
+									: {
+											toggleChecked: adUnit.status === 'ACTIVE',
+											toggleOnChange: value => {
+												adUnit.status = value ? 'ACTIVE' : 'INACTIVE';
+												updateAdUnit( adUnit );
+											},
+									  } ) }
+								description={ () => (
+									<span>
+										{ gamConnectionStatus.can_connect && adUnit.is_legacy ? (
+											<i>{ __( 'Legacy ad unit.', 'newspack' ) }</i>
```

Comment:
Can we add a separator after this flag?

<img width="208" alt="Screen Shot 2021-06-09 at 4 09 41 PM" src="https://user-images.githubusercontent.com/2230142/121436055-1ce6ae00-c93d-11eb-87e2-7d8679a7ac2a.png">

Expected:

_Legacy ad unit_ | Sizes: `300x250`

Reply:
Fixed in bb65fde8276a12fd3e56c6c98c1a0cf5380c9ff0